README.md

```diff
@@ -1 +1,57 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+>  To Do List Project For Microverse By Bwalya Nonde.
+
+
+> This project is on HTML CSS and JAVASCRIPT. this is a to do list structure project, which i implimented the earlier mentioned languages, and added the webpack.
+To do List
+
+
+
+## Built With
+
+- html
+- css
+- javascript
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+git clone git@github.com:BwalyaNonde/to-do-list.git
+cd Add_and_remove
+npm install
+npm run build
+npm start
+Then check http://localhost:8080/
+
+
+
+
+## Author
+👤 **Bwalya Nonde* 
+
+- GitHub: [@BwalyaNonde](https://github.com/BwalyaNonde)
+- Twitter: [nonde_bwalya](https://twitter.com/nonde_bwalya)
+- LinkedIn: [Bwalya Nonde](https://www.linkedin.com/in/bwalya-nonde-5865601a9/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Please don't forget to add MIT.md license file 📃

Reply:
@Rustamxon7 Thank you for approving my project 👍.
Thanks for the reminder.